Client/Users/index.ts

```diff
@@ -67,8 +67,11 @@ export class Users extends Collection<model.UserResponse, model.UserSearchReques
 	): Promise<model.PasswordValidateResponse | model.ErrorResponse> {
 		return await this.connection.post<model.PasswordValidateResponse>(`users/password/check`, request)
 	}
-	async changePassword(request: model.PasswordChangeRequest): Promise<model.UserResponse | model.ErrorResponse> {
-		return await this.connection.put<model.UserResponse>(`users/password`, request)
+	async changePassword(
+		request: model.PasswordChangeRequest,
+		otp?: number
+	): Promise<model.UserResponse | model.ErrorResponse> {
+		return await this.connection.put<model.UserResponse>(`users/password`, request, undefined, { "x-otp": otp })
```

Comment:
should this only write the headers object if otp isnt undefined

otherwise itll be sending junk